pages/take-action.vue

```diff
@@ -124,6 +133,32 @@ export default Vue.extend({
 .v-application--wrap {
   justify-content: space-between;
 }
+
+.Take-Action-Title{
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 24px;
+  line-height: 33px;
+  /* identical to box height */
+
+  text-align: center;
+
+  color: #2A6465; 
+}
+
+.Take-Action-Subheading{
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 25px;
+  /* identical to box height */
+
+
+  color: #000000;
```

Comment:
nit: use sass variable
```suggestion
  color: $black-1;
```